tests/qos/test_buffer.py

```diff
@@ -734,8 +779,6 @@ def test_change_speed_cable(duthosts, rand_one_dut_hostname, conn_graph_facts, p
                             pool_oid,
                             pool_size = original_pool_size,
                             shp_size = original_shp_size,
-                            old_speed = original_speed,
-                            new_speed = speed_to_test,
```

Comment:
why is this removed

Reply:
As we discussed, we don't need this anymore.
The motivation for passing old/new speed is to adjust the `xon` of the port accordingly in order to calculate the pool size. Now we use the number of lanes which won't be changed during the test.